extensions/kotlin/README.md

```diff
@@ -0,0 +1,13 @@
+# OpenTelemetry Kotlin Extension
+
+To enable kotlin support, you need to add the `io.opentelemetry:opentelemetry-extension-kotlin` dependency.
```

Comment:
maybe goes unsaid, but as for the info/example below I personally had to dig into stackoverflow comments/responses to find and also came up in micronaut community forum on how to do.

Reply:
This is documented in the [README](https://github.com/open-telemetry/opentelemetry-java#api-extensions). I have a mild preference for having each piece of info live in one place to reduce the chance of it going out of date.

Reply:
Yep I agree